<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerVoIP</class>
 <widget class="QMainWindow" name="ManagerVoIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManagerVoIP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MovedBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>401</width>
      <height>291</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>250</height>
     </size>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>72</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>a</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>b</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>c</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="mnuFile_Settings"/>
    <addaction name="separator"/>
    <addaction name="mnuFile_Exit"/>
   </widget>
   <widget class="QMenu" name="mnuPhones">
    <property name="title">
     <string>Aoutoprovision</string>
    </property>
    <addaction name="mnuPhones_Yealink"/>
    <addaction name="mnuPhones_Cisco"/>
    <addaction name="mnuPhones_LinkSys"/>
    <addaction name="separator"/>
    <addaction name="mnuPhones_Wizard"/>
   </widget>
   <widget class="QMenu" name="mnuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="mnuHelp_Help"/>
    <addaction name="mnuHelp_website"/>
    <addaction name="mnuHelp_CheckUpdate"/>
    <addaction name="separator"/>
    <addaction name="mnuHelp_About"/>
    <addaction name="mnuHelp_AboutQt"/>
   </widget>
   <widget class="QMenu" name="mnuAsteriskConfig">
    <property name="title">
     <string>Asterisk</string>
    </property>
    <addaction name="actionCDR"/>
    <addaction name="actionConfig"/>
   </widget>
   <widget class="QMenu" name="mnuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="mnuView_ShowMenu"/>
    <addaction name="mnuView_ShowToolBar"/>
    <addaction name="mnuView_ShowStatusBar"/>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="mnuAsteriskConfig"/>
   <addaction name="mnuView"/>
   <addaction name="mnuPhones"/>
   <addaction name="mnuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mnuFile_Exit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mnuFile_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="mnuFile_Settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="mnuPhones_Yealink">
   <property name="text">
    <string>Yealink</string>
   </property>
  </action>
  <action name="mnuPhones_Cisco">
   <property name="text">
    <string>Cisco</string>
   </property>
  </action>
  <action name="mnuPhones_LinkSys">
   <property name="text">
    <string>LinkSys</string>
   </property>
  </action>
  <action name="mnuPhones_Wizard">
   <property name="text">
    <string>Wizard</string>
   </property>
  </action>
  <action name="mnuHelp_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="mnuHelp_Help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="mnuHelp_website">
   <property name="text">
    <string>Go to web-site</string>
   </property>
  </action>
  <action name="mnuHelp_CheckUpdate">
   <property name="text">
    <string>Check Update</string>
   </property>
  </action>
  <action name="mnuHelp_AboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionCDR">
   <property name="text">
    <string>Call Detail Record</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="mnuView_ShowMenu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Menu</string>
   </property>
  </action>
  <action name="mnuView_ShowToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show ToolBar</string>
   </property>
  </action>
  <action name="mnuView_ShowStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show StatusBar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MovedBox</class>
   <extends>QGroupBox</extends>
   <header>movedbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
